### PR TITLE
chore_: Remove Goerli code

### LINKED ISSUE #20273: wallet. - remove goerli support
We no longer need support for testnet Goerli, remove all uses of it 👍

### PR DESCRIPTION
**Pull request by smohamedjavid**
fixes #20273

### Summary

This PR removes the goerli-related code from the mobile repo. 

### Review notes

We removed just the Goerli toggle from the old advanced settings (https://github.com/status-im/status-mobile/pull/20875) 3 months back when Goerli ceased to exist.

The PR will NOT bring any functional changes to the app. The app will work as before.

### Testing notes

A complete regression testing on the app is appreciated.

**Note:** The testnet mode (sepolia) is enabled by default on Dev, PR, and E2E builds as before. This PR doesn't change that. 

### Platforms

- Android
- iOS

##### Functional

- chats / community permission
- wallet / transactions
- dapps 
- new account
- networks

status: ready

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,6 +1,5 @@
 CACHED_WEBVIEWS_ENABLED=1
 DEBUG_WEBVIEW=1
-DEFAULT_NETWORK=mainnet_rpc
```
Comment (smohamedjavid) [inline .env:3]: We no longer use the default network env and send it to status-go.

File: modules/react-native-status/ios/RCTStatus/LogManager.m [hunk 2]
```diff
@@ -34,25 +34,25 @@ @implementation LogManager
...
+//     NSString *goerliNetworkDirPath = @"ethereum/goerli_rpc";
+// #endif
+
+    // NSURL *networkDir = [rootUrl URLByAppendingPathComponent:networkDirPath];
+    // NSURL *originalGethLogsFile = [networkDir URLByAppendingPathComponent:@"geth.log"];
+    // NSURL *gethLogsFile = [logsFolderName URLByAppendingPathComponent:@"mainnet_geth.log"];
+
+    // NSURL *goerliNetworkDir = [rootUrl URLByAppendingPathComponent:goerliNetworkDirPath];
+    // NSURL *goerliGethLogsFile = [goerliNetworkDir URLByAppendingPathComponent:@"geth.log"];
+    // NSURL *goerliLogsFile = [logsFolderName URLByAppendingPathComponent:@"goerli_geth.log"];
```
Comment (smohamedjavid) [inline modules/react-native-status/ios/RCTStatus/LogManager.m]: We no longer use network-based logs (used in v1). Now (in v2), Irrespective of the network, the status-go logs are output to the `geth.log` file.

Surprisingly, only iOS have this old code, so I have commented.

cc @siddarthkay

File: src/status_im/contexts/wallet/common/utils/networks.cljs [hunk 3]
```diff
@@ -10,21 +10,15 @@
...
-   constants/optimism-goerli-chain-id  constants/optimism-network-name
    constants/optimism-sepolia-chain-id constants/optimism-network-name
    constants/arbitrum-mainnet-chain-id constants/arbitrum-network-name
-   constants/arbitrum-goerli-chain-id  constants/arbitrum-network-name
    constants/arbitrum-sepolia-chain-id constants/arbitrum-network-name})
 
 (defn- get-chain-id
-  [{:keys [mainnet-chain-id sepolia-chain-id goerli-chain-id testnet-enabled? goerli-enabled?]}]
+  [{:keys [mainnet-chain-id sepolia-chain-id testnet-enabled?]}]
   (cond
```
Comment (vkjr) [inline src/status_im/contexts/wallet/common/utils/networks.cljs]: Can be a simple `if`

**Comment by vkjr** [comment]
Nice cleanup! Thank you!

**Review by vkjr — APPROVED** [review]

**Review by jakubgs — APPROVED** [review]
LGTM

File: modules/react-native-status/ios/RCTStatus/LogManager.m (hunk 2 above)
Comment (briansztamfater) [reply modules/react-native-status/ios/RCTStatus/LogManager.m]: I think we can safely delete this code

File: src/utils/ethereum/eip/eip681_test.cljs [hunk 4]
```diff
@@ -111,9 +111,9 @@
                                                            :name "Status Network Token"
                                                            :symbol :SNT
                                                            :decimals 18}}
-   :goerli  {"0x3d6afaa395c31fcd391fe3d562e75fe9e8ec7e6a"
+   :sepolia {"0xE452027cdEF746c7Cd3DB31CB700428b16cD8E51"
```
Comment (briansztamfater) [inline src/utils/ethereum/eip/eip681_test.cljs:114]: nit: You can move the STT contract address to a constant. If Sepolia testnet becomes obsolete in the future, we can just update that constanta.

**Review by briansztamfater — APPROVED** [review]
🚀

File: modules/react-native-status/ios/RCTStatus/LogManager.m (hunk 2 above)
Comment (siddarthkay) [reply modules/react-native-status/ios/RCTStatus/LogManager.m]: @smohamedjavid : yeah we can delete this code.

**Review by siddarthkay — APPROVED** [review]
just delete commented code in `modules/react-native-status/ios/RCTStatus/LogManager.m` and rest looks good.
Thanks

**Review by mohsen-ghafouri — APPROVED** [review]
👌

**Review by clauxx — APPROVED** [review]

**Comment by VolodLytvynenko** [comment]
hi @smohamedjavid thank you for PR. No issues from my side. PR is ready to be merged

**Pull request merged**